docs/_static/searchtester.js

```diff
@@ -0,0 +1,156 @@
+const { switchMap } = rxjs;
+const nucliaResult = document.querySelector("nuclia-search-results");
+const shadowRoot = nucliaResult.shadowRoot;
+
+const nuclia = new window.NucliaSDK.Nuclia({
+  backend: "https://nuclia.cloud/api",
+  zone: "europe-1",
+  knowledgeBox: "62407006-2711-4631-9c03-761d156de289",
+});

```

Comment:
I accidentally pushed the kb token here 
I think we should make another secret for `knowledgeBox`

Reply:
@justdaksh Don't worry, that's not a token, that's the kb id, and that's not really a secret because the kb id is in the URL of the requests made by the widget, so it is necessarily publicly visible.